book/extend-admin.rst

```diff
@@ -209,6 +210,9 @@ shows a controller doing what has just been described.
     use Sulu\Component\Rest\RestHelperInterface;
     use Symfony\Component\HttpFoundation\Response;
 
+    /**
+     * @RouteResource("Event")
```

Comment:
I think this should not be uppercase:

```
@RouteResource("event")
```

e.g.:

```
@RouteResource("event-registration")
```

Reply:
the `FOSRestBundle` will lower case this automatically. if you dont add any annotation, the bundle will use `Event` (capitalized) too: https://github.com/handcraftedinthealps/RestRoutingBundle/blob/a9a11287e5b04235c18b1b46ab8dc69408191892/Routing/Loader/Reader/RestControllerReader.php#L78-L98

Reply:
yeah but it would be better represents that its a resource key if you set it to lowercase

Reply:
What happens if we use the resource key here does it work correctly?

```
@RouteResource(Event::RESOURCE_KEY)
```

Reply:
yes, looks like that is working 🙂

Reply:
but that will lead to the same problem if you use a `_` in your resource key 🙈

Reply:
🙈